src/diffusers/models/unet_1d.py

```diff
@@ -70,8 +70,9 @@ def __init__(
         self.sample_size = sample_size
 
         # time
-        self.time_proj = GaussianFourierProjection(embedding_size=8)
-        del self.time_proj.W
+        self.time_proj = GaussianFourierProjection(
```

Comment:
Should we format this like in the 2d class?

Also, no embedding after projection?

https://github.com/huggingface/diffusers/blob/6cbb73b57e6b679d0de8ae47e80d50874ad5fb14/src/diffusers/models/unet_2d.py#L101-L109

Reply:
Think as soon as we have more than GaussianFourier, let's do it - before it's maybe not necessary

Reply:
The RL unet1d is different, was prepping for that.